core/src/nucleus/mod.rs

```diff
@@ -44,6 +46,12 @@ impl ZomeFnCall {
             parameters: parameters.to_string(),
         }
     }
+
+    pub fn same_as(&self, fn_call: &ZomeFnCall) -> bool {
```

Comment:
can this be a `PartialEq` implementation?

Reply:
Not really since we are not comparing every field.
I renamed the function hoping it makes this clearer.